e2e/keywords/deployment.resource

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation    Deployment Keywords
+
+Library    Collections
+Library    ../libs/keywords/common_keywords.py
```

Comment:
TODO: Suggest adding `libs` to the lib path, so that we can import our libraries from the python lib paths directly.

Reply:
Chinya, please help create a ticket for this improvement.